### ISSUE TITLE
Bones roll value is getting incorrectly set?

### ISSUE THREAD

**Issue by NickaNak** [BadDogSkyrim/PyNifly#112]
When importing a skeleton the importer sets a bone's roll differently to what the Nif Tools Blender addon does,
I know this sounds like a silly issue but it completely breaks any animation made on a rig imported via your plugin plugin and then exported to game via the FBX to HKX tool

I don't know if your plugin or the nif tools plugin has the correct value but the FBX to HKX exporter only works with the Nif Tools Blender addon

Thanks for your time

**Comment by BadDogSkyrim** [comment]
Thanks for the report. 

I *think* the bone roll correctly matches what's in the nif, but I agree it should match whatever the rest of the ecosystem expects. If you could send me a blender file with the bones set up correctly that would be a help.

**Comment by NickaNak** [comment]
Hey sorry for the late reply Just remembered this thread
Here's the base female skeleton
https://drive.google.com/file/d/1phVmf-bgL20OQtNMiuRGl-e4KZ5TgStu/view?usp=sharing
Here's the beast female skeleton
https://drive.google.com/file/d/1g4VANoLa0r4WLSPORj4502CXHHbpESYW/view?usp=sharing
Here's the XP Female skeleton, don't know if it'll help but included it in case
https://drive.google.com/file/d/1wtQb8wVUunOSdwCfqfRwBd4AoB7R_t8k/view?usp=sharing

These were imported using the most recent Nif tools blender plugin, I've tested these roll values and they do work in game,
I haven't included all the creature skeletons either as that's just silly 
Thanks again for looking into this!

**Comment by BadDogSkyrim** [comment]
Yeah, thanks. I don't need all the skeletons (probably)--one bone will likely be enough.

**Comment by BadDogSkyrim** [comment]
Picking this up to work on now. Do you understand what the niftools version is doing? What I'm seeing is a random -90deg roll thrown into the NPC, NPC Root, NPC COM, and NPC Pelvis bones, which I think are all 0. It could be that there's a -90deg roll added to everything for Reasons, but I dunno.

**Comment by NickaNak** [comment]
I'm really sorry but I don't know why or how the Niftools importer is doing the adjustments on the rolls

**Comment by BadDogSkyrim** [comment]
K thx. Asking over on their channel.

**Comment by BadDogSkyrim** [comment]
Status: I haven't been able to figure out what niftools is doing with bone transforms. Since they work with animations, it may be that they're doing something special exactly so animations will work. I did some work with animations on my side and got results that look like bone transforms are messed up (so far as Anton's animation tools are concerned).

I'm calling this an enhancement and back-burnering it for now.

**Comment by NickaNak** [comment]
No worries! And thanks for looking into this
